statics/js/components/capture-form.js

```diff
@@ -105,6 +112,9 @@ Vue.component('capture-form', {
       userQuery: "",
       mode: "selection",
       visible: false,
+      captureType: "",
+      nodeType: "",
+      typeAllowed: true,
```

Comment:
I would rather put it as false by default, because we can allow the user the select the type only if we know which nodes are selected.

Reply:
done